### ISSUE TITLE
Support relative coordinates

### ISSUE THREAD

**Issue by HarelM** [elastic/spritezero#4]
I have the following file - which is a svg file generated from inkscape:
[cattle_grid.svg.txt](https://github.com/mapbox/spritezero/files/4305089/cattle_grid.svg.txt)
It seems that the path letters are lowercase which causes this library to generate an empty sdf.
I'm also not sure about the view box as my file has a viewbox that is different from the width and height which might cause problems, not sure though, but it seems that this library ignores it...
I'm not sure I know svg enough to make a PR.
Thanks for this great addition!

**Comment by HarelM** [comment]
Looking at the code, the following line should've solve this issue, not sure why it doesn't though...
https://github.com/elastic/spritezero/blob/2d5963920d294dea5b477016c4c794429143e7a0/index.js#L162

**Comment by HarelM** [comment]
Nevermind me, the problem is with the reading of the SVG correctly.
I'll send a PR once I'll fix it.

**Comment by HarelM** [comment]
Ok, Found the right sequence of calls, I'll see if I can send the PR tonight.
The following is the code to handle transform, scale, viewbox. it should replace the code wirtten here:
https://github.com/elastic/spritezero/blob/2d5963920d294dea5b477016c4c794429143e7a0/index.js#L161L175
```js

          var svgPath = new SvgPath(path.getAttribute('d')).scale(options.pixelRatio);
		  
          var parent = path.parentElement;
          while (parent.tagName === 'G' || parent.tagName === 'g') {
            if (parent.getAttribute('transform')) {
              svgPath = svgPath.transform(parent.getAttribute('transform'));
            }
            parent = parent.parentElement;
          }
          if (el.getAttribute('viewBox')) {
            var viewBox = el.getAttribute('viewBox').split(/\s+/);
            svgPath = svgPath.translate(-viewBox[0], -viewBox[1])
				.scale(w/(viewBox[2] - viewBox[0]), h/(viewBox[3] - viewBox[1]));
          }
          svgPath = svgPath.abs().unshort().unarc();
```

**Comment by HarelM** [comment]
I've sent a PR to fix this issue: PR #5.
